### PR TITLE
Clean up, some documentation, and fixes for slider app to work with the figure class

### PR DESCRIPTION
**Pull request by rothnic**
Some quick fixes to get the sliders app working again, plus some additional clean up.

### CODE REVIEW

**Comment by damianavila** [comment]
For reference, this is linked with #1703.
@rothnic do you have plans to add more thing to this PR... or you want to iterate with follow up PR later?
If this is ready, I will label it properly so people can review it soon.

File: examples/app/sliders_applet/sliders_app.py [hunk 1]
```diff
@@ -85,10 +94,20 @@ def create(cls):
         return obj
 
     def setup_events(self):
+        """Attaches the on_change event to the value property of the widget.
+
+        The callback is set to the input_change method of this app.
+
+        :return: None
```
Comment (bryevdv) [inline examples/app/sliders_applet/sliders_app.py]: small change: we use Google style docstrings consistently, that is what `sphinx-napoleon` expects. So this should just be:
```
    Returns:
        None
````
Here is a full reference: http://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_google.html

**Comment by bryevdv** [comment]
I'm really not sure how updating the api got missed in the deprecations PR, thanks @rothnic!

**Comment by rothnic** [comment]
This would be it. I'm working with some of these apps, and if I find anything I'll just provide a fix to get them back in a working state.

This should now be good on the docstring formatting. Too bad pycharm doesn't support that style, there appears to be an [open issue](https://youtrack.jetbrains.com/issue/PY-9795). I couldn't find anything in its settings to lean towards the newline for the closing parens, but I did find some settings I needed to change otherwise. I could post the screenshots of the settings in the docs if that would be of use.

**Comment by damianavila** [comment]
@rothnic do you plan to add more things? So far, this LGTM... I restarted TravisCI because of a spurious run...

**Comment by bryevdv** [comment]
LGTM too, I will mention as an aside that the other `app` examples are similarly broken and need to be updated after the deprecations were removed.

**Comment by damianavila** [comment]
>  I will mention as an aside that the other app examples are similarly broken and need to be updated after the deprecations were removed.

@rothnic, do you want to try fixing the other examples? If you can not do it... there is no problem, we can take care of them, but if you can do it, it would be awesome, and you could do it in this same PR :wink:

**Comment by damianavila** [comment]
I relaunched TravisCI, since you open #1757 for the Crossfilter, I assume this is done... I will merge it in a few minutes. Thanks again for your contribution.

**Pull request merged**